scrapy/mail.py

```diff
@@ -135,3 +152,37 @@ def _sendmail(self, to_addrs, msg):
             reactor.connectTCP(self.smtphost, self.smtpport, factory)
 
         return d
+
+
+class SESMailSender(BaseMailSender):
+
+    def __init__(self, aws_access_key, aws_secret_key, mailfrom='scrapy@localhost', debug=False):
+        self.aws_access_key = aws_access_key
+        self.aws_secret_key = aws_secret_key
+        self.mailfrom = mailfrom
+        self.debug = debug
+
+    @classmethod
+    def from_settings(cls, settings):
+        return cls(
+            settings['MAIL_SES_AWS_ACCESS_KEY'],
+            settings['MAIL_SES_AWS_SECRET_KEY'],
+            settings['MAIL_FROM']
```

Comment:
I'd prefer to use kwargs here as well, for robustness

Reply:
Not sure if I understood the way you are thinking to use kwargs in this situation. Are you talking in something like this?

```
    def __init__(self, debug=False, **kwargs):
        self.aws_access_key = kwargs.get('MAIL_SES_AWS_ACCESS_KEY')
        self.aws_secret_key = kwargs.get('MAIL_SES_AWS_SECRET_KEY')
        self.mailfrom = kwargs.get('MAIL_FROM', 'scrapy@localhost')
        self.debug = debug

    @classmethod
    def from_settings(cls, settings):
        return cls(**settings)
```

Reply:
I meant 

```py
return cls(
    aws_access_key=settings['MAIL_SES_AWS_ACCESS_KEY'],
    aws_secret_key=settings['MAIL_SES_AWS_SECRET_KEY'],
    mailfrom=settings['MAIL_FROM']
)
```